src/test-utils/index.tsx

```diff
@@ -189,3 +189,31 @@ export const openSSRPage = ({
 
 export const setRootFontSize = (px: number): Promise<void> =>
     page.$eval('html', (e, px) => e.setAttribute('style', `font-size: ${px}px;`), px);
+
+export const waitFor = <T,>(
```

Comment:
Why don't you use the waitFor from RTL?

https://testing-library.com/docs/dom-testing-library/api-async/#waitfor